### PR TITLE
Implemented menu

### LINKED ISSUE #20: Implement menu
Add hamburger menu following [the design](https://www.figma.com/file/4qqHCvcrTdm6kqwJHDUde3/Astoria-Digital-Design-System?node-id=651%3A1010)

### PR DESCRIPTION
**Pull request by olesia-ag**
**mobile (not sure if the menu should cover the header?)**:
Screen Shot 2021-02-24 at 2 55 34 PM
**desktop**:
Screen Shot 2021-02-24 at 2 37 54 PM
Screen Shot 2021-02-24 at 2 37 42 PM

### CODE REVIEW

**Review by aniagaska — COMMENTED** [review]
It can replace the header! It would be good if it didn't overlap on the rest of the content of the site -- could the subheading and the rest of the content of the page get pushed down by it?

**Comment by olesia-ag** [comment]
@aniagaska I can just make it the exact same height as the header on mobile, and make a top margin for the subheading ("Visualizing the process..") bigger so that the menu doesn't cover it on the desktop view - if I push it only when the menu is opened (if I understood what you meant), I think it might look like the text on the page 'jumps' up and down

**Comment by aniagaska** [comment]
@olesia-ag that makes sense to me! I wouldn't want the transition to be jumpy. Thanks Olesia!

File: browser/src/components/Toolbar/NavigationItems/NavigationItems.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const NavigationItems = () => {
+
+	return (
+		<div  className="navigation_items">
```
Comment (bronzehedwick) [inline browser/src/components/Toolbar/NavigationItems/NavigationItems.js]: Can we fix the indentation levels here?

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
This is great, thanks Olesia! Had just one minor formatting comment and then I think we'll be good to merge in.

**Review by bronzehedwick — APPROVED** [review]

**Pull request merged**